<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Uncodemy" failures="1" tests="1" name="Testing-Sanity(failed)" time="5.467" errors="0" timestamp="2024-02-04T15:56:34 IST">
  <testcase classname="day1.SeleniumTest" name="f" time="4.911">
    <failure type="java.lang.AssertionError" message="expected [GitHub: Let’s build from here · GitHub] but found [Let&amp;apos;s build from here]">
      <![CDATA[java.lang.AssertionError: expected [GitHub: Let’s build from here · GitHub] but found [Let's build from here]
at day1.SeleniumTest.f(SeleniumTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Testing-Sanity(failed) -->
